tests/kafkatest/services/console_consumer.py

```diff
@@ -123,6 +123,7 @@ def __init__(self, context, num_nodes, kafka, topic, group_id="test-consumer-gro
         self.from_beginning = from_beginning
         self.message_validator = message_validator
         self.messages_consumed = {idx: [] for idx in range(1, num_nodes + 1)}
+        self.node_indexes_clean_shutdown = set()
```

Comment:
Haha, fair enough. Another option would be to actually use the node objects in the set. Anyway, just a nitpick.